authors/apps/profiles/serializers.py

```diff
@@ -20,12 +21,23 @@ def get_image(self, obj):
         ###the avatar that will be displayed in case user hasnt uploaded
         #profile picture yet.
 
+    def get_following(self, instance):
+        request = self.context.get('request', None)
+
+        if not request:
+            return False
+
+        follower = request.user.profile
+        followee = instance
+        
+        return follower.is_following(followee)
```

Comment:
I would suggest you change the name followee to something more descriptive like some_one_being_followed in yo own word

Reply:
Plus Also consider adding info for person testing the application that they should install the newly added requirements since I saw you updated your requirements.txt